tests/functional/adapter/materialized_view_tests/test_materialized_view.py

```diff
@@ -0,0 +1,64 @@
+from typing import List
+from dbt.contracts.relation import RelationType
+from dbt.tests.adapter.materialized_view.base import (
+    assert_model_exists_and_is_correct_type,
+    run_model,
+    # get_row_count,
+    # insert_record,
+)
+from tests.functional.adapter.materialized_view_tests.fixtures import (
+    BigQueryBasicBase,
+)
+
+
+RecordSet = List[tuple]
+
+
+class TestBasic(BigQueryBasicBase):
+    def test_relation_is_materialized_view_on_initial_creation(self, project):
+        assert_model_exists_and_is_correct_type(
+            project, "base_materialized_view", RelationType.MaterializedView
+        )
+        assert_model_exists_and_is_correct_type(project, "base_table", RelationType.Table)
+
+    def test_relation_is_materialized_view_when_rerun(self, project):
+        run_model("base_materialized_view")
+        assert_model_exists_and_is_correct_type(
+            project, "base_materialized_view", RelationType.MaterializedView
+        )
+
+    # def test_relation_is_materialized_view_on_full_refresh(self, project):
```

Comment:
I use `@pytest.mark.skip("<message for why we're skipping>")` in these scenarios. It's more readable because you get the formatting back, tests that should be skipped will still likely get updated since the syntax checker will hit it, and there's a built in message as to why these tests are skipped.